Source/Moya.swift

```diff
@@ -133,11 +137,16 @@ public class MoyaProvider<Target: MoyaTarget> {
                 cancellableToken.innerCancellable = self.stubRequest(target, request: request, completion: completion, endpoint: endpoint, stubBehavior: stubBehavior)
             }
         }
-
+        
         requestClosure(endpoint, performNetworking)
-
+        
         return cancellableToken
     }
+
+    /// Convenience request-making method that takes no extra endpoint parameters
+    public func request(target: Target, completion: Moya.Completion) -> Cancellable {
```

Comment:
True. I just hadn't done that initially to prevent breaking existing code that uses Moya, whose parameter-less calls to `request()` will need updated to explicitly be `request(parameters: nil)` without this bridging function. If you're saying the breaking change is still okay with you, I will change it though.